app/views/effective/classifieds/index.html.haml

```diff
@@ -1,9 +1,38 @@
+- classifieds = ::Effective::Classified.classifieds(user: current_user).paginate(page: params[:page], per_page: 12)
+
 = render 'layout' do
   .resource-buttons
     = link_to "Submit New #{et(Effective::Classified)}", effective_classifieds.new_classified_wizard_path, class: 'btn btn-primary'
 
-  = card do
-    - if @datatable.present?(self)
-      = render_datatable(@datatable, simple: true)
-    - else
-      %p There are no active #{etsd(Effective::Classified)}. When there are, we'll show them here.
+  - if classifieds.present?
+    .actions.d-flex.flex-row-reverse.my-3
+      - ['list', 'grid'].each do |view_type|
+        = link_to effective_classifieds.classifieds_path(view_type: view_type, page: params[:page]), class: "btn #{params[:view_type].blank? && view_type == 'list' && 'btn-primary'} #{params[:view_type] == view_type ? 'btn-primary' : 'btn-default'} btn-sm" do
+          %span.px-1
+            = icon(view_type)
+            = view_type.capitalize
+
+    .row
+      - classifieds.each do |classified|
+        .mb-3{ class: params[:view_type] == 'grid' ? 'col-12 col-sm-6 col-md-4 col-xl-3' : 'col-12' }
+          .card.h-100
+            .card-body.d-flex.flex-column
+              %h5.card-title
+                = link_to classified.title, effective_classifieds.classified_path(classified), class: 'stretched-link text-decoration-none'
+              %p.card-text.pt-2
+                = classified.organization
+                %br
+                = classified.location
+              %p.card-text.mt-auto
+                %small.text-muted
+                  %strong Published
+                  = classified.start_on&.strftime("%b %d, %Y")
+                  %br
+                  %strong Closing
+                  = classified.end_on&.strftime("%b %d, %Y")
+
+    .row.d-flex.justify-content-center
+      = paginate(classifieds, per_page: 12)
```

Comment:
We have this `config.per_page` variable in effective_classifieds.rb.  Let's change this into an integer and use `EffectiveClassifieds.per_page` here